processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -554,12 +562,22 @@ func (c *WatchClient) extractPodAttributes(pod *api_v1.Pod) map[string]string {
 		}
 	}
 
+	formatterLabel := "k8s.pod.labels.%s"
```

Comment:
Maybe do those formats constants and move on top of file for easier tracking?

Reply:
I guess we can use directly the semconv library here? Since I expect the new lib will be released sooner than this PR will get merged. Added it here, just to showcase the functionality is working as expected